vimeo-networking/src/test/java/com/vimeo/networking/model/HlsVideoFileTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Vimeo
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.vimeo.networking.model;
+
+import com.vimeo.networking.Utils;
+
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link HlsVideoFile}.
+ * <p>
+ * Created by zetterstromk on 8/3/17.
+ */
+public class HlsVideoFileTest {
+
+    @Test
+    public void verifyTypeAdapterWasGenerated() throws Exception {
+        Utils.verifyTypeAdapterGeneration(HlsVideoFile.class);
```

Comment:
That does feel a bit like an optimization - and in the interest of keeping this PR scoped towards just removing deprecated methods/classes (in the cleanest way possible), I'd say it could be ticketed separately and put in the vimeo-networking backlog (to be prioritized by TPMs). Maybe just quoting Mo's original suggestion in the description (so we don't lose the intent).